docs/application/native/guides/machine-learning/overview.md

```diff
@@ -0,0 +1,14 @@
+# Machine-Learining
+
+
+Machine-learning features introduce how you can handle machine learning frameworks, like tensorflow, tensorflow-lite, with the construction of data pipeline.
+
+You can use the following machine-learning features in your native applications:
+
+- [Machine Learning Inference C-API](machine-learning-inference.md)
+
+  You can construct the data pipeline which contains the machine learning model filter. The pipeline can be assembled and controlled with various elements of [gstreamer](https://gstreamer.freedesktop.org/). 
```

Comment:
Please make this small change, to maintain parallelism with the related document:

You can easily construct and efficiently execute data stream pipelines, which contain the machine learning model filter. The pipeline can be assembled and controlled with various elements of [GStreamer](https://gstreamer.freedesktop.org/).